packages/office-ui-fabric-react/src/components/Link/Link.doc.tsx

```diff
@@ -21,15 +21,11 @@ export const LinkPageProps: IDocPageProps = {
           root: {
             selectors: {
               '.ms-Link': {
-                color: theme!.palette.themePrimary,
                 margin: 0,
                 padding: 0,
                 overflow: 'inherit',
                 textOverflow: 'inherit',
                 selectors: {
-                  ':active, :hover, :active:hover': {
-                    color: theme!.palette.themeDarker,
-                  },
                   ':focus': {
```

Comment:
Shoudn't this be removed too if you're removing the other state overrides? Potentially same for the disabled state.